packages/blade/codemods/brand-refresh/transformers/__tests__/migrate-colors.test.ts

```diff
@@ -61,7 +61,7 @@ it('should update token values contextually', async () => {
               <MapIcon color="interactive.icon.primary.normal" />
               <MapIcon color="surface.text.onSea.onSubtle" />
               <MapIcon color="interactive.icon.primary.normal" />
-              <MapIcon color="interactive.icon.gray.default" />
+              <MapIcon color="interactive.icon.gray.normal" />
```

Comment:
Snapshot update for an older commit https://github.com/razorpay/blade/pull/1846/commits/c1c78b5285210d9a15f10e6636afa59c9dc08d42